dashboard/test/factories/factories.rb

```diff
@@ -1058,6 +1058,17 @@
     level_source {create :level_source, level: level}
   end
 
+  factory :skill do
+    description {"Declares variables with conventional names"}
+    concept {"Variables"}
+    evaluation_criteria {"Does the student's work on this level demonstrate the skill?"}
```

Comment:
not sure which PR will go in first, but once you add `key` to `Skill`, you will probably want to add it here as a `sequence` like we have just below for unit name.